app/screens/channel_info/extra/extra.tsx

```diff
@@ -131,25 +227,32 @@ const Extra = ({channelId, createdAt, createdBy, customStatus, header, isCustomS
                     id='channel_info.header'
                     defaultMessage='Header:'
                     style={styles.extraHeading}
-                    testID='channel_info.extra.header'
-                />
-                <Markdown
-                    channelId={channelId}
-                    baseTextStyle={styles.header}
-                    blockStyles={blockStyles}
-                    disableBlockQuote={true}
-                    disableCodeBlock={true}
-                    disableGallery={true}
-                    disableHeading={true}
-                    disableTables={true}
-                    location={Screens.CHANNEL_INFO}
-                    textStyles={textStyles}
-                    layoutHeight={48}
-                    layoutWidth={100}
-                    theme={theme}
-                    imagesMetadata={headerMetadata}
-                    value={header}
+                    testID={headerTestId}
                 />
+                <TouchableWithFeedback
+                    type='opacity'
+                    activeOpacity={0.8}
+                    onLongPress={() => handleLongPress()}
```

Comment:
We are passing here a new function on every render. I think I understand why you don't do just `onLongPress={handleLongPress}` since onLongPress will send an event, and will mess with the logic of `handleLongPress`. If that is the case, maybe another `useCallback` would work here.
```
const touchableHandleLongPress = useCallback(() => handleLongPress(), [handleLongPress]);
```